### DISCUSSION TITLE
Auto hide Rust Analyzer error window

### DISCUSSION THREAD

**Discussion by TN19N** [zed-industries/zed#19461] [Feature Requests]
Hi, I just tried Zed on Linux, and it's awesome! I enabled auto-save in Zed, and I’m wondering if there’s an option to automatically hide the Rust Analyzer error window. It updates each time I make a change, but once I fix everything, I expect it to hide automatically. It’s quite annoying to have to click to close it each time I change something.

Thanks!

![Screenshot from 2024-10-19 15-02-01](https://github.com/user-attachments/assets/d05e8860-6e5c-4cbc-9717-2f7a9c9a82e6)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
